windows/power_operation_scripts/power_operation_scripts.yml

```diff
@@ -29,6 +30,10 @@
           ansible.builtin.set_fact:
             vmware_toolbox_cmd_path: "C:\\Program Files\\VMware\\VMware Tools\\VMwareToolboxCmd.exe"
 
+        - name: "Set facts of VMware Tools log files"
```

Comment:
Task name can be changed to `Set fact of collecting VMware Tools config and log files`.